tests/integration/targets/info_test/vars.yml

```diff
@@ -7,4 +7,9 @@ info_validate_certs: false
 
 mock_file: "info_test"
 
-info_cluster: true
+# workaround till simulator fixed
```

Comment:
what is the issue with the simulator? not sure it's clear 
Is there a ticket to fix it?

Reply:
see [here](https://github.com/redhat-cop/cloud.vmware_ops/tree/main/roles/cluster_settings#requirements) regarding pyvmomi version

Reply:
also, it seems that the simulator info tests are always skipped for your integration tests
so I think it is something to do with how you setup the podman container

Reply:
Can you add more details to the comment? when it will be removed? who will remove it?

Reply:
done